esp/src/src-react/components/Helpers.tsx

```diff
@@ -33,6 +33,9 @@ function getURL(item: HelperRow, option) {
         case "ComponentLog":
             params = "/WUFile/" + item.Type + "?Wuid=" + item.workunit.Wuid + "&Name=" + item.Orig.Name + "&Type=" + item.Orig.Type;
             break;
+        case "postmortem":
+            params = "/WUFile/" + item.Type + "?Wuid=" + item.workunit.Wuid + "&Name=" + encodeURIComponent(item.Orig.Name) + "&Type=" + item.Orig.Type;
+            break;
```

Comment:
The `encodeURIComponent` should probably be added for everything after the "?"?

Reply:
I just added the encodeURIComponent(). Please review.